src/durableClient/DurableClient.ts

```diff
@@ -492,8 +493,24 @@ export class DurableClient implements types.DurableClient {
                 .replace(this.instanceIdPlaceholder, instanceIdPath);
         }
 
+        // Get the current active span
+        const currentSpan = OpenTelemetryApi.trace.getSpan(OpenTelemetryApi.context.active());
+        let traceParent = "";
+        let traceState = "";
+        const headers: Record<string, string> = {};
+        if (currentSpan) {
+            const spanContext = currentSpan.spanContext();
+            traceParent = `00-${spanContext.traceId}-${
```

Comment:
Do we have to manually create the `traceParent` and `traceState` here? It looks like `propagation.inject` can do this for us ([example ](https://opentelemetry.io/docs/languages/js/propagation/#generic-example) here)

Reply:
Done